src/index.html

```diff
@@ -22,6 +22,397 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="containerHeader">
+      <a
+        href="#"
+        class="containerHeader__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active">Apple</a>
```

Comment:
Interactive elements like links should have an 'href' attribute to ensure proper accessibility and semantics. Add an 'href' to this anchor tag.